src/Splat.Common.Test/ViewTwo.cs

```diff
@@ -12,7 +12,9 @@ namespace Splat.Common.Test;
 /// </summary>
 /// <seealso cref="ViewModelTwo" />
 [ExcludeFromCodeCoverage]
+#pragma warning disable CA1515 // Consider making public types internal
```

Comment:
I'd disable CA1515 for test projects at the very least. It's more meant for projects where its a fully enclosed project.